docs/components/dartpad/progress_indicators/theme/main.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
```

Comment:
Please redo the theme screenshot after making updates to this demo

Reply:
Done